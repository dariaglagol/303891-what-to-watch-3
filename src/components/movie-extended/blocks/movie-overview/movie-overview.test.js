import React from "react";
import renderer from "react-test-renderer";
import MovieOverview from "./movie-overview";

const MovieOverviewMock = {
  director: `director`,
  starring: `starring`,
  runTime: 113,
  genre: `genre`,
  releaseDate: `releaseDate`
};

it(`Movie overview render`, () => {
  const {director, starring, runTime, genre, releaseDate} = MovieOverviewMock;

  const movieOverviewComponent = renderer
    .create(
        <MovieOverview
          director={director}
          starring={starring}
          runTime={runTime}
          genre={genre}
          releaseDate={releaseDate}
        />
    ).toJSON();

  expect(movieOverviewComponent).toMatchSnapshot();
});
