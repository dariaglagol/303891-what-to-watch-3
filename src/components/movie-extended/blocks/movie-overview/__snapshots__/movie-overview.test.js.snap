// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie overview render 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
         givenPromoMovie director
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
         givenPromoMovie starring
         
        <br />
        givenPromoMovie starring
         
        <br />
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        113
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        givenPromoMovie genre
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2020
      </span>
    </p>
  </div>
</div>
`;
